### DISCUSSION TITLE
img src component prop is url encoded

### DISCUSSION THREAD

**Discussion by piotrblasiak** [remarkjs/remark#1268] [Q&A] [answered]
I implemented a custom img component and noticed that the src prop is url encoded, so I need to to decodeURIComponent first before using it to get the original value. Is this on purpose and if so, why?

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @piotrblasiak! 👋 

> Is this on purpose and if so, why?

It is on purpose, the `src` attribute is a URL (ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#src), and URLs need to be URL encoded.
If you need to get the unsafe form of the URL, the `decodeURIComponent` function can be used.
